npc/custom/jobmaster.txt

```diff
@@ -305,7 +306,7 @@ function	Job_Menu	{
 
 // Executes the actual jobchange and closes.
 function	Job_Change	{
-	function Get_Platinum;
+	callfunc "F_GetPlatinumSkills";
```

Comment:
calling the function at the start would result of getting the Platinum skills for anyone who talk to the npc without `.Platinum = true;` so removing this line(309) from here would be the best option

Reply:
thanks very much, edited as your request

Reply:
`function Get_Platinum;`
^ For clarifications: This (old) line not a function call. It's just a forward declaration, that this function exist, so we can call it before the function is defined.

Reply:
okay jeybla so what should i do? revert back using forward function?